src/onnxruntime.cc

```diff
@@ -858,7 +858,7 @@ ModelInstanceState::ValidateBooleanSequenceControl(
   std::string tensor_datatype;
   RETURN_IF_ERROR(GetBooleanSequenceControlProperties(
       sequence_batching, model_state_->Name(), control_kind, required,
-      &tensor_name, &tensor_datatype, nullptr, nullptr, nullptr, nullptr,
+      &tensor_name, &tensor_datatype, nullptr, nullptr, nullptr, nullptr
```

Comment:
Does this code still compiles? The comma at the end is still needed.

Reply:
Good catch! Just corrected my serious typo. I only changed this line to make it compatible with my local triton-inference-server version... This updated code now can be compiled